sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/SignalingClient.java

```diff
@@ -0,0 +1,20 @@
+package com.azure.android.communication.chat.signaling;
+
+public interface SignalingClient {
```

Comment:
any reason to have the public contract `SignalingClient`? are we expecting chat module to ship with more clients (in addition to `CommunicationSignalingClient`) that implements this?

Reply:
It needs to be public so that chatClient can call methods of it.